scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixEvaluation.java

```diff
@@ -4,7 +4,7 @@
 
 public interface ScalafixEvaluation {
     /**
-     * @return boolean true if scalafix has successfully been evaluated on all files configured and no ScalafixError has been found
+     * @return boolean true if scalafix has successfully been evaluated on all files configured with no error except LintErrors.
```

Comment:
should the `ScalafixFileEvaluation#isSuccessful` implementation & doc be updated as well?